openjdk/playlist.xml

```diff
@@ -1438,6 +1438,40 @@
 			<group>openjdk</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>jdk_tools_openj9_DynamicLoadWarningTest</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/adoptium/aqa-tests/issues/3232#issuecomment-1032587426</comment>
+				<platform>x86-64_alpine-linux|aarch64_alpine-linux</platform>
+			</disable>
+		</disables>
```

Comment:
This is not needed as we only run jdk_tools_openj9_DynamicLoadWarningTest against openj9/ibm.

Reply:
Removed the `disables`.